docs/usage.rst

```diff
@@ -11,7 +11,7 @@ then it should be available on pypi.
 * `bidict <https://github.com/jab/bidict>`_
 * `binaryornot <https://github.com/audreyr/binaryornot>`_
 * `brotlipy <https://github.com/python-hyper/brotlipy>`_
-* `chardet <https://pypi.python.org/pypi/chardet>`_
+* :pypi:`chardet`
```

Comment:
I wonder if we should add a similar helper for github (not a requested change, just a thought)

Reply:
The way custom roles work, the default link text has to be the whole slug, so adding a new bit of custom config doesn't do much:

    `brotlipy <https://github.com/python-hyper/brotlipy>`_
    :github:`brotlipy <python-hyper/brotlipy>`

and it hardly seemed worth the trouble.